### PR TITLE
Updates unturned events

### PR DESCRIPTION
**Pull request by IAmSilK**
- Fixes incorrect inheritances
- Removes unnecessary events
- Refactors namespace for better consistency
- Added UnturnedPlayerGestureUpdatedEvent

### CODE REVIEW

File: unturned/OpenMod.Unturned/Players/Movement/Events/PlayerMovementEventsListener.cs [hunk 1]
```diff
@@ -31,13 +31,13 @@ public override void Unsubscribe()
         public override void SubscribePlayer(Player player)
         {
             player.stance.onStanceUpdated += () => OnStanceUpdated(player);
-            player.animator.onGestureUpdated += () => OnGestureUpdated(player);
+            player.animator.onGestureUpdated += (gesture) => OnGestureUpdated(player, gesture);
```
Comment (DiFFoZ) [inline unturned/OpenMod.Unturned/Players/Movement/Events/PlayerMovementEventsListener.cs:34]: `onGestureUpdated` event does not include a punch gesture. See: https://github.com/SmartlyDressedGames/Unturned-3.x-Community/issues/1927

Comment (DiFFoZ) [reply unturned/OpenMod.Unturned/Players/Movement/Events/PlayerMovementEventsListener.cs:34]: And also many gestures not included: Facepalm, Point, Wave, and etc..

**Comment by Trojaner** [comment]
I sadly didn't see @DiFFoZ's comment when merging, so it would be appreciated if these could be fixed too

File: unturned/OpenMod.Unturned/Players/Movement/Events/PlayerMovementEventsListener.cs (hunk 1 above)
Comment (IAmSilK) [reply unturned/OpenMod.Unturned/Players/Movement/Events/PlayerMovementEventsListener.cs:34]: Fixed by #203

**Pull request merged**